src/themes/five/src/components/ProductForm/index.js

```diff
@@ -244,17 +244,14 @@ const ProductOptionsUI = (props) => {
     if (!isErrors) {
       return
     }
-    const myElement = document.getElementsByClassName('error')[0]
-    const productContainer = document.getElementsByClassName('product-container')[0]
-    if (!myElement || !productContainer) {
-      return
-    }
-    let topPos = myElement.offsetTop - productContainer.offsetTop
-    if (windowSize.width <= 768) {
-      const productImage = document.getElementById('product_image')
-      topPos = topPos + (myElement.offsetTop < productImage.clientHeight ? productImage.clientHeight : 0)
+    const productContainer = document.getElementsByClassName('popup-dialog')[0]
+    const errorCount = document.getElementsByClassName('error').length
+    let myElement = document.getElementsByClassName('error')[0]
+    if (errorCount > 1) {
+      myElement = document.getElementsByClassName('error')[1]
     }
-    scrollTo(productContainer, topPos, 200)
+    myElement.scrollIntoView(true)
```

Comment:
add validation in case it's  `myElement` undefined

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining